src/main/java/dynamo/RidesDao.java

```diff
@@ -28,19 +26,24 @@ public RidesDao() {
     }
 
     // Todo: Implement insert, update, read, readAll, delete
-    public void insert() {
-    	try {
-            System.out.println("Adding a new item...");
-            PutItemOutcome outcome = table.putItem(new Item().withPrimaryKey("year", year, "title", title).withMap("info", infoMap));
-
-            System.out.println("PutItem succeeded:\n" + outcome.getPutItemResult());
-
-        }
-        catch (Exception e) {
-            System.err.println("Unable to add item: " + year + " " + title);
-            System.err.println(e.getMessage());
-        }
+    public void insert(ClientRequest request)

```

Comment:
I think we might have to change these a bit, but let's not worry about it for now.